week-2/implement/contains.js

```diff
@@ -1,3 +1,5 @@
-function contains() {}
+function contains(object, key) {
+  return !object ? false : object.hasOwnProperty(key);
+}
 
 module.exports = contains;
```

Comment:
Great work!

Reply:
Thank you mate, for your time and assitance.